nav2_map_server/CMakeLists.txt

```diff
@@ -21,21 +21,37 @@ include_directories(include)
 
 set(map_server_executable map_server)
 
-set(map_saver_executable map_saver)
+set(map_saver_cli_executable map_saver_cli)
+
+set(map_saver_server_executable map_saver_server)
 
 add_executable(${map_server_executable}
-  src/main.cpp)
+  src/map_server_main.cpp)
+
+add_executable(${map_saver_cli_executable}
+  src/map_saver_cli_main.cpp)
+
+add_executable(${map_saver_server_executable}
+  src/map_saver_server_main.cpp)
 
-add_executable(${map_saver_executable}
-  src/map_saver_cli.cpp)
+set(mapio_library_name mapio)
 
 set(library_name ${map_server_executable}_core)
 
+add_library(${mapio_library_name} SHARED
+  src/map_mode.cpp
+  src/mapio.cpp)
```

Comment:
lets rename this to `map_io.cpp` to be consistent with other files

Reply:
File was renamed (https://github.com/ros-planning/navigation2/commit/89ece9672dc847161607acc658b63fe2bafb90a7#diff-2a1a2abf02d6011dd7803064feb30d95R43)